src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -2083,6 +2056,19 @@ void Schedule::handleGRUPTREE( const DeckKeyword& keyword, size_t currentStep, c
         return *group_ptr;
     }
 
+    std::vector<Group2*> Schedule::getGroups2(size_t timeStep) const {
+        std::vector<Group2*> groupsAtTimestep;
+        if (timeStep >= this->m_timeMap.size())
+            throw std::invalid_argument("timeStep argument beyond the length of the simulation");
+
+        for (const auto& dynamic_pair : this->groups) {
+            auto& group_ptr = dynamic_pair.second.get(timeStep);
+            if (group_ptr)
```

Comment:
Look ma - no braces!

Reply:
Yes I don't like it, but I try to preserve existing style. The function is identical to getWells2 apart from replacing `this->wells_static` with `this->groups`.